spec/readfile_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rspec'
+require_relative '../lib/readfile.rb'
+
+describe 'Number of words' do
+    it 'get words return the proper number' do
```

Comment:
2 spaces. https://github.com/bbatsov/ruby-style-guide

Check out this great site on rspec best practices.
www.betterspecs.org